src/yamlfix/services.py

```diff
@@ -17,17 +17,22 @@
 Files = Union[Tuple[TextIOWrapper], List[str]]
 
 
-def fix_files(files: Files) -> Optional[str]:
+def fix_files(files: Files, dry_run: bool = False) -> Optional[str]:
```

Comment:
I think you forgot to change the function signature, the return value is no longer `Optional[str]`

Reply:
Indeed, fixed!